### DISCUSSION TITLE
How self-promotional _should_ we be?

### DISCUSSION THREAD

**Discussion by richb-hanover** [PRQL/prql#1669] [General]
I am so pleased with the [One year of PRQL](https://prql-lang.org/posts/2023-02-02-one-year/) post. It really shows that we are a real project, making real progress toward our goal.

One comment to discuss if we feel like navel-gazing: [@aljaz wrote:](https://github.com/PRQL/prql/pull/1661#discussion_r1095675042) 

> It seems too promotional to quote ourselves.

I would suggest that the entire point of the post was to be self-promotional. To provide links to stuff we have written is neither hyperbolic nor peddling snake oil. (Or maybe you should just quote that "authority" over at [Random Neurons Firing](https://randomneuronsfiring.com/format-pretty-reports-in-prql/) :-)

No need for further discussion today, but I do want to remind everyone that it's difficult for us to over-communicate to the outside world...

**Comment by max-sixty** [comment]
Always appreciate the moral questions!

I think @aljazerzen more meant re "quoting ourselves" there is that quotes are often used to bring credibility to a message. But quoting ourselves saying "Isn't PRQL so great" is self-referential. And possibly assigning too much status to ourselves...

**Reply by aljazerzen** [reply]
Yup, that's what I meant.

**Comment by richb-hanover** [comment]
OK. Points accepted.

What "promotional" posts are we thinking about next? Thanks again.

**Comment by vanillajonathan** [comment]
I don't think you should quote yourself on how great PRQL is because it is unprofessional, dishonest and lacks integrity. I think it would be better with a honest approach that objectively points out the benefits, features and strengths of PRQL which I believe would speak for itself when people see it and think "Oh it can do this, and that, wow that's nice, I really like that, that would make my life easier!".

PRQL have ambitions and ambitions are good, so I think the ambitions should be mentioned too, like "_this_ is what is implemented so far, and _this_ is what we're aiming to have in the future" which would further bring excitement.

**Comment by max-sixty** [comment]
Locking this — I think the question has been answered. If anyone has any feedback about anything that's actually been written, that's very welcome indeed.
